pages/strategy.md

```diff
@@ -0,0 +1,73 @@
+
+Our most important goal at this stage is being able to give our current students a timely mentoring experience, and be able to cope with further growth without overwhelming mentors.
+
+We aim to achieve this goal by the end of 2019.
+
+## Strategy
+
+**Acquire and retain more mentors by making mentoring rewarding and enjoyable**
+
+Currently, mentoring can be both fulfilling and frustrating. It can be very rewarding to help a student that wants to learn. On the flipside, it can be repetitive, students can be rude, there are few positive feedback loops, and it can feel too much like a job rather than a hobby. In order to retain our existing mentors, and to acquire new ones, we must make mentoring more enjoyable and more rewarding. We must make our mentors feel like that are part of something bigger than them, and that they are contributing to something valuable. A good parallel is Wikipedia, where thousands of people contribute (from once a year, to dozens of times per day) because they feel like they are making a tangible difference to something that matters, and because the process itself is straight-forward and (strangely) rewarding.
+
+## Tactics
+
+### Restructure tracks for student enjoyment and ease of mentoring
+
+Some solutions are fun to mentor, some are extremely challenging. For example, solutions that require maths lead to mentors trying to tutor someone on algebra, not on coding. Other solutions can be solved in such a variety of ways that the mentor ends up debating the right algorithm, not teaching the idioms of the language.
+
+We aim to fix this by determining the optimal set and order of core exercises within a track, which will provide a sensible progression for students and be fun to review for mentors.
+
+_Update: We have called this this the [Track Anatomy Project](https://exercism.io/blog/track-anatomy-project). In our prototype Track (Ruby), the result of the Track Anatomy Project saw a average reduction in waiting time from one week to half a day, and reduced the queue from hundreds to ~30 at any time. It also saw a doubling in the number of submissions per week and dramatically increased mentor retention, with the most active weekly number mentors of any track. Encoding this into a framework and applying it to all tracks should have a dramatic positively effect on mentor enjoyment._
+
+### Automatically analyze common solutions
+
+The majority of submissions are common. It is rare for a student to submit a solution that no-one else has ever submitted. As part of restructing the track, we aim to enforce this as a constraint, guiding tracks to have core exercises that are usually solved with one of a few common approaches. We aim to utilize this consistency to provide automated approval and/or feedback to students for the majority of submissions to core exercises. This has the following benefits:
```

Comment:
I think my previous comment got lost in history somewhere (I'm not seeing it, anyway).

> We aim to enforce this as constraint

It's not clear what "this" points to.